src/systems/eu/implementations/react/packages/site-header/SiteHeader.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from '@ecl/eu-react-component-icon/Icon';
+import Logo from '@ecl/eu-react-component-logo/Logo';
+import SearchForm from '@ecl/eu-react-component-search-form/SearchForm';
+
+const SiteHeader = ({
+  logo,
+  languageSelector,
+  searchForm,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, {
+    'ecl-site-header': true,
+  });
+
+  return (
+    <div className={classNames} {...props}>
+      <div className="ecl-site-header__container ecl-container">
+        <div className="ecl-site-header__banner">
+          <a
+            className="ecl-site-header__logo ecl-link ecl-link--standalone"
+            href={logo.href}
+          >
+            <Logo title={logo.title} alt={logo.alt} language={logo.language} />
+          </a>
+
+          <div className="ecl-site-header__selector">
+            <a
```

Comment:
why not use `<Link ... />`?

Reply:
because Link do not allow anything else from simple text for now